src/python/pants/engine/legacy/source_mapper.py

```diff
@@ -48,20 +49,27 @@ def _unique_dirs_for_sources(self, sources):
   def target_addresses_for_source(self, source):
     return list(self.iter_target_addresses_for_sources([source]))
 
-  def _iter_owned_files_from_hydrated_target(self, legacy_target):
+  def _match_source(self, source, fileset):
+    if os.path.exists(source):
+      return fileset.matches(source)
+    else:
+      return matches_filespec(source, fileset.filespec)
+
+  def _own_source(self, source, legacy_target):
     """Given a `HydratedTarget` instance, yield all files owned by the target."""
```

Comment:
Comment out of date. I think this method is probably `_owns_source` since it returns a boolean?